### PR TITLE
This PR  fixes #  The what is swarm page on bee-docs

### PR DESCRIPTION
**Pull request by erickNjuki**
1. Paraphrased the underlay network , overlay network and the  different layers.
2. Removed some words on first sentences in order avoid repetition.
3. Introduced un-ordered list. it is good practice to differentiate the lists.
4. Also made grammatical corrections.

### CODE REVIEW

**Review by bennehc — COMMENTED** [review]
Removed repetition on first sentences

I dont think  this change is needed.  Currently it reads well and provides more context.

File: docs/learn/technology/what-is-swarm.md [hunk 1]
```diff
@@ -11,59 +11,56 @@ Swarm is peer-to-peer network of nodes which work together to provide decentrali
...
-5. Channel security – The channel provides identity verification and implements
-encrypted and authenticated transport resisting man in the middle attacks.
-6. Protocol multiplexing – The underlay network service can accommodate several
-protocols running on the same connection. 
-7. Delivery guarantees – Protocol messages have guaranteed delivery, i.e. delivery
-failures due to network problems result in direct error response. Order of delivery
-of messages within each protocol is guaranteed. 
-8. Serialisation – The protocol message construction supports arbitrary data structure
-serialisation conventions.
+Swarm serves as an underlay transport by establishing connections between nodes in the network and routing data between them. This provides a low-level communication channel that enables nodes to communicate with each other directly, without relying on any centralised infrastructure.
```
Comment (NoahMaizels) [inline docs/learn/technology/what-is-swarm.md:21]: Swarm includes both underlay and overlay, so it is not really accurate to say that Swarm itself serves as an underlay transport. It includes both these layers.

File: docs/learn/technology/what-is-swarm.md [hunk 2]
```diff
@@ -11,59 +11,56 @@ Swarm is peer-to-peer network of nodes which work together to provide decentrali
...
+- Serialisation – The protocol message construction supports arbitrary data structure
+  serialisation conventions.
 
 As the [libp2p](https://libp2p.io/) library meets all these requirements it has been used to build the Swarm underlay network.
 
-
 ### 2. Overlay Network
 
-The second part of Swarm is an overlay network with protocols powering the [Distributed Immutable Store of Chunks (DISC)](/docs/learn/technology/DISC/). This layer is responsible for storing and retrieving data in a decentralised and secure manner.
+Swarm is an overlay network with protocols powering the [Distributed Immutable Store of Chunks (DISC)](/docs/learn/technology/DISC/). This layer is responsible for storing and retrieving data in a decentralised and secure manner.
```
Comment (NoahMaizels) [inline docs/learn/technology/what-is-swarm.md:48]: Same issue as above about underlay, Swarm is not an overlay network, Swarm _includes_ an overlay network

File: docs/learn/technology/what-is-swarm.md [hunk 3]
```diff
@@ -11,59 +11,56 @@ Swarm is peer-to-peer network of nodes which work together to provide decentrali
...
 
 Each chunk has a fixed size of 4kb and is distributed across the network using the DISC model. Each chunk has a unique address taken from the same namespace as the network node addresses that allows it to be located and retrieved by other nodes in the network.
 
 Swarm's distributed immutable storage provides several benefits, including data redundancy, tamper-proofing, and fault tolerance. Because data is stored across multiple nodes in the network, it can be retrieved even if some nodes fail or go offline.
 
 ### 3. Data Access Layer
 
-The third part of Swarm is a component that provides high-level data access and defines APIs for base-layer features. This layer is responsible for providing an easy-to-use interface for developers to interact with Swarm's underlying storage and communication infrastructure.
+Swarm is a component that provides high-level data access and defines APIs for base-layer features. This layer is responsible for providing an easy-to-use interface for developers to interact with Swarm's underlying storage and communication infrastructure.
```
Comment (NoahMaizels) [inline docs/learn/technology/what-is-swarm.md:60]: Same issue as the other two sections, Swarm _includes_ a component, it is not the component itself

**Review by NoahMaizels — CHANGES_REQUESTED** [review]
A lot of the wording edits look good and thank you for correcting the typo! There are some conceptual problems with how the layers are described, if you can refer to the comments I left and address those it would be great!

**Comment by NoahMaizels** [comment]
Haven't received a response to my comments for several weeks so I'm closing this PR

**Pull request closed without merging**